airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java

```diff
@@ -1130,6 +1135,11 @@ private void writeStandardSync(final List<StandardSync> configs, final DSLContex
                     io.airbyte.db.instance.configs.jooq.generated.enums.StatusType.class).orElseThrow())
             .set(CONNECTION.SCHEDULE, JSONB.valueOf(Jsons.serialize(standardSync.getSchedule())))
             .set(CONNECTION.MANUAL, standardSync.getManual())
```

Comment:
Do we plan to remove the manual flag one day, It sounds confusing to had a SCHEDULE_TYPE that won't include MANUAL.

Reply:
Just check the enum and manual is part of it. Should we validate in the persistence that the manual column and the enum value are synchronized?

Reply:
Added validation; please take a look.